src/chrome/content/rules/Curse.xml

```diff
@@ -3,36 +3,187 @@
 
 		- bukkit.org.xml
 
+	TLS not supported on:
+	- forums.wowace.com (only supported by cloudflare frontend, not curse backend)
```

Comment:
Back when i created this PR you received a cloudflare error page stating that they can't connect securely to the original server.